Sources/Plasma/NucleusLib/pnAsyncCoreExe/Private/Nt/pnAceNt.cpp

```diff
@@ -298,7 +298,7 @@ void INtConnCompleteOperation (NtObject * ntObj) {
         break;
 
         default:
-            LogMsg(kLogError, "NtConnCompleteOp %p %u", ntObj, ntObj->ioType);
+            LogMsg(kLogError, "NtConnCompleteOp {x} {}", (uintptr_t)ntObj, ntObj->ioType);
```

Comment:
One additional comment:  `%p` in printf actually equates to `%#x` more closely, so you may want to change the pointers to `{#x}` instead of just `{x}`